container.go

```diff
@@ -410,6 +448,8 @@ func (c *Container) start() error {
 		return err
 	}
 
+	c.storeMounts()
```

Comment:
No, I think this should be done at the container/pod level from having every agent implementation having to do this.
@sameo What do you think, you were mentioning the same right?